cores/epoxy/Client.h

```diff
@@ -44,15 +42,16 @@ class Client: public Stream {
         virtual uint8_t connected() = 0;
         virtual operator bool() = 0;
     protected:
-		/* does not compile
+
         uint8_t* rawIPAddress(IPAddress& addr) {
             return addr.raw_address();
         }
-        const uint8_t* rawIPAddress(const IPAddress& addr) {
+
+        #if defined(EPOXY_CORE_ESP8266)
+        const uint8_t* rawIPAddress(IPAddress& addr) const {
```

Comment:
... but this method exists only in ESP8266. However, I think you have a typo. This should be
`const uint8_t* rawIPAddress(const IPAddress& addr) {` as it was originally, because that's the signature in the ESP8266 core: https://github.com/esp8266/Arduino/blob/master/cores/esp8266/Client.h#L45C25-L45C61